### PR TITLE
Fix null pointer exception when finishing building a decoration

### PR DESCRIPTION
**Pull request by xavierh**
Closes #

# Changes proposed in this pull request:
-
-
-

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/colony/WorkManager.java [hunk 1]
```diff
@@ -73,7 +73,10 @@ public void removeWorkOrder(final int orderId)
             final WorkOrderBuild wob = (WorkOrderBuild)workOrder;
             final AbstractBuilding building = colony.getBuilding(wob.getBuildingLocation());
             ConstructionTapeHelper.removeConstructionTape(wob, colony.getWorld());
-            building.markDirty();
+            if (building == null)
+            {
+                building.markDirty();
```
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/colony/WorkManager.java]: BLOCKER NullPointerException might be thrown as 'building' is nullable here [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2259)

**Comment by MinecoloniesSonar** [comment]
SonarQube analysis reported 1 issue
* ![BLOCKER][BLOCKER] 1 blocker

Watch the comments in this conversation to review them.

[BLOCKER]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER'

File: src/main/java/com/minecolonies/coremod/colony/WorkManager.java (hunk 1 above)
Comment (MinecoloniesSonar) [inline src/main/java/com/minecolonies/coremod/colony/WorkManager.java]: BLOCKER NullPointerException might be thrown as 'building' is nullable here [rule](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2259)

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/692/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Comment by Raycoms** [comment]
:shipit:

**Comment by marvin-bitterlich** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/Minecolonies/minecolonies/pull-request/692/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Minecolonies/minecolonies)

**Pull request merged**